__init__.py

```diff
@@ -253,6 +256,14 @@ def handle_system_reboot(self, _):
 
     def handle_system_shutdown(self, _):
         subprocess.call(["/usr/bin/systemctl", "poweroff"])
+        
+    def handle_remove_namespace(self, message):
+        get_skill_namespace = message.data.get("skill_id", "")
+        if get_skill_namespace:
+            self.bus.emit(Message("gui.clear.namespace",
+                                  {"__from": get_skill_namespace}))
+        self.resting_screen.cancel_override()
```

Comment:
Will this cancel any override screen regardless of whether it matches the `skill_id` of the namespace we're clearing?

Eg could the following scenario occur?
- Skill A is triggered, performing some action
- Skill B is triggered and overrides idle
- Skill A finishes and clears it's namespace
- Skill B's idle override gets unintentionally cancelled

Reply:
the idle / resting screen implementation to me atleast doesn't seem to account for namespaces or maintain any list of skills that have an override, It seems to only apply the override logic on the current page being displayed so Even if Skill A is triggered and is performing some action, Skill B by just being triggered after that will probably cause it to override the previous Skill A idle override, the cancel_override() will then only cancel Skill B override because that is what the resting screen currently knows

Reply:
Yeah, definitely outside the scope of this PR, but I wonder if we need to look at that. 

I'll create a new issue for that and merge this in.

Thanks :)